platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
No, org.exoplatform:maven-parent-pom:16-SNAPSHOT is not patt of platform-ui project, it is in https://github.com/exoplatform/maven-parent-pom/.
But do you mean that it does not compile if you don't add the version of the lesscss-maven-plugin ? It should actually.